src/views/forms/TradeForm/AdvancedTradeOptions.tsx

```diff
@@ -144,8 +159,12 @@ export const AdvancedTradeOptions = () => {
             )}
             {showReduceOnly && (
               <Checkbox
-                checked={(reduceOnly && !reduceOnlyTooltip) || false}
-                disabled={!!reduceOnlyTooltip}
+                checked={
+                  (reduceOnly && !reduceOnlyTooltip) ||
+                  complianceState === ComplianceStates.CLOSE_ONLY ||
```

Comment:
hmm I wonder if it'd be useful to extract this as a helper somewhere? Side note, does this mean that the SL/TP dialog (and close positions on the positions table) should be enabled for `ComplianceStates.CLOSE_ONLY`?

Reply:
Hmmm good question. They can close their position without having to use SL/TP right? In that case I'm thinking we can just hide the SL/TP dialogs.

Reply:
> They can close their position without having to use SL/TP right 
yup! I agree - let's just hide em